BuildPackagesFromLayoutDirectory.cmd

```diff
@@ -41,8 +41,8 @@ goto :Exit
 xcopy /Y %LayoutForSigningDirectory%\net452\%1 %BinaryOutputDirectory%\net452\
 
 :: For .NET core, .exes are renamed to .dlls due to packaging conventions
-xcopy /Y %LayoutForSigningDirectory%\netcoreapp2.0\%~n1.dll %BinaryOutputDirectory%\netcoreapp2.0\
-xcopy /Y %LayoutForSigningDirectory%\netstandard2.0\%~n1.dll %BinaryOutputDirectory%\netstandard2.0\
+xcopy /Y %LayoutForSigningDirectory%\netcoreapp2.1\%~n1.dll %BinaryOutputDirectory%\netcoreapp2.1\
```

Comment:
>2.1 [](start = 47, length = 3)

No urgency, but we could use this opportunity to upgrade to 3.1. I remembered it wrong in the past, 2.2 is out of support and 2.1 is supported until January, 2021 [https://dotnet.microsoft.com/platform/support/policy/dotnet-core](https://dotnet.microsoft.com/platform/support/policy/dotnet-core)

Reply:
Got it, yes.

---
In reply to: [377832840](https://github.com/microsoft/sarif-sdk/pull/1750#discussion_r377832840) [](ancestors = 377832840)